README.md

```diff
@@ -1,11 +1,85 @@
 [![Build Status](https://travis-ci.com/SUSE-Enceladus/public-cloud-info-client.svg?branch=master)](https://travis-ci.com/SUSE-Enceladus/public-cloud-info-client)
 
-pint
-====
+# Introduction
 
 pint (Public Cloud INformation Tracker) is a command line client to access
 the REST API provided by the SUSE Public Cloud Information Service. The
 information service provides data about the images SUSE publishes in various
 public cloud frameworks (Amazon, Google, Microsoft) as well as the
 servers that make up the SUSE maintained and operated update infrastructure
 in those frameworks.
+
+## Quick Start
+
+There are various ways to install the Pint Client:
+
+### Install On SUSE Linux using rpm package  
+
+```bash
+  $ sudo zypper in python3-susepubliccloudinfo
+```
+
+### Install using pip on non-SUSE distros, e.g. Debian/Ubuntu or RHEL/CentOS. 
+
+```bash   
+  # Clone the repo
+  $ git clone https://github.com/SUSE-Enceladus/public-cloud-info-client
+  $ cd public-cloud-info-client
+
+  # create the Python 3.6 development virtual environment.
+  $ virtualenv pint_venv --python=python3
+  
+  # activate the development virtual environment
+  $ source pint_venv/bin/activate
```

Comment:
Depends on the shell being used, should probably be pointed out that this works for bash. venv creation also generates a activate.sh file for csh and derivatives.